src/homework6/classes.py

```diff
@@ -0,0 +1,143 @@
+"""Create simulator from real life. This can be booking room in hotel,
+visit to casino, visit to bar. Create 3-4 objects, that can describe situation
+Objects should contain attributes and methods to simulate some use cases.
+Completed program should print object states, it actions (methods)
+and objects interaction."""
+
+
+class Vehicle(object):
+    """Describes vehicle with model, key and size.
+
+    Key is unique for each vehicle.
+    Size shows how much space vehicle will occupy in parking lot.
+    """
+    def __init__(self, model, size):
+        self.model = model
+        self.size = size
+        self.key = id(self)
+        self.parked = False
+
+    def __str__(self):
+        return self.model
+
+
+class Driver(object):
+    """Describes person who owns vehicle and key for it.
+
+    He can buy, sell, park and pick up vehicle from parking lot.
+    """
+    def __init__(self, name, *vehicles):
+        self.name = name
+        if vehicles:
+            self.vehicles = {vehicle.key: vehicle for vehicle in vehicles}
+        else:
```

Comment:
we can do it without else: if vehicles would be empty, our dict comprehension will return {}